alf/environments/alf_wrappers.py

```diff
@@ -944,3 +944,49 @@ def _step(self, action):
             time_step = time_step._replace(discount=np.float32(0))
         self._prev_lives = lives
         return time_step
+
+
+@alf.configurable
+class TemporallyCorrelatedNoiseWrapper(AlfEnvironmentBaseWrapper):
+    """Adding temporally correlated noise to actions.
+    Reference:
+    ::
+        Swamy et al. Causal Imitation Learning under Temporally Correlated Noise, arXiv:2202.01312
+    """
+
+    def __init__(self, env, sigma=1.0, past_weight=0.5, current_weight=0.1):
```

Comment:
Seems that there are more than enough arguments :) 
with `sigma`, `past_weight` and `current_weight = 1 - past_weight` it would be simpler and achieve everything that can be achieved by 3 parameters.

Reply:
In the paper the authors used two weights that are not sum to one. Updated the arguments to support both cases as suggested.

Reply:
Sorry I was just curious why the authors choose to have 3 parameters, when they can just adjust `sigma` assuming weights summing to 1. Thanks!

Reply:
The previous redundant way maybe a bit intuitive when adjusting each factors but is unnecessary. Now reduced the argument list.